### DISCUSSION TITLE
Feature request: play a sound when adjusting individual app volume levels

### DISCUSSION THREAD

**Discussion by xezrunner** [File-New-Project/EarTrumpet#722] [Ideas]
When adjusting the global volume, the Asterisk sound effect is played, giving you a sense for how loud the volume is.
This is not the case when adjusting individual app volumes.

As an example, I would like to change ShareX's screenshot notification sound volume to be quieter. I do not have a sense of scale for how loud the lower levels will be.

Changing the volume multiple times for different apps would benefit from having a sound effect being played.

I would recommend adding a toggle in the EarTrumpet settings for playing a preview sound, both applying to global and individual volume adjusting.

**Comment by riverar** [comment]
Yep, related to #425. Gets a bit complicated as EarTrumpet isn't an audio _client_ (like sndvol, etc.) so can't play on non-default devices at this time. Will need to fix this at some point.
